graylog2-web-interface/src/views/components/datatable/DataTableEntry.tsx

```diff
@@ -122,14 +127,17 @@ const DataTableEntry = ({ columnPivots, fields, series, columnPivotValues, value
     <tr className={`fields-row ${classes}`}>
       {columns.map(({ field, value, path, source }, idx) => {
         const key = `${activeQuery}-${field}=${value}-${idx}`;
+        const fieldName = getFieldNameFromTrace({ series, name: field });
```

Comment:
Can we use `columnNameToField` here?

Reply:
`getFieldNameFromTrace` gives me the field name and `columnNameToField` gives the series function. I can reuse `columnNameToField` as a part of `getFieldNameFromTrace` to get the series function but not as a replacement.

Reply:
We could use it in conjunction with `parseSeries` to get the field from it.